Doc/library/codecs.rst

```diff
@@ -1443,10 +1443,26 @@ to :class:`bytes` mappings. They are not supported by :meth:`bytes.decode`
 | uu_codec             | uu               | Convert the operand using    |                              |
 |                      |                  | uuencode.                    |                              |
 +----------------------+------------------+------------------------------+------------------------------+
+| escape_codec         | escape           | Encode and decode byte       | :func:`codecs.escape_encode` |
+|                      |                  | sequences using escape       | /                            |
+|                      |                  | sequences, similar to        | :func:`codecs.escape_decode` |
+|                      |                  | :func:`repr` of bytes.       |                              |
++----------------------+------------------+------------------------------+------------------------------+
```

Comment:
I don't think this is correct. The codec itself (string_escape) was removed after the transition to Python 3. There is no `escape_codec` to be documented, only the functions which could be used to create a new `bytes_escape` codec, but that's probably for a separate issue to take care of.